components/IntroDiagram.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import { Box, Paragraph, Diagram, Stack, Text } from "grommet";
+
+import { TestDesktop, Edit, Code, System } from 'grommet-icons';
+
+const thickness = 'xsmall';
+const color = '#6FFFB0';
```

Comment:
We try to use colors that are not hardcoded and part of the HPE design-system theme https://github.com/grommet/grommet-theme-hpe